New package: BiGSTARS v2.0.2

JuliaRegistrator opened the pull request:
- Registering package: BiGSTARS
- Repository: https://github.com/subhk/BiGSTARS.jl
- Created by: @subhk
- Version: v2.0.2
- Commit: e2a93a2e32712434f575c237af0aedf09cfaee90
- Reviewed by: @subhk
- Reference: https://github.com/subhk/BiGSTARS.jl/issues/9#issuecomment-3114938321
- Description: A Julia toolkit for bi-global linear stability analysis for geophysical flows using Chebyshev-Fourier spectral collocation method.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835986850970ccc7c25489f5434d2c9a0297442efb1c5a421502e677d802c10ad298d7e9173c03b7482dd617bf8498ceadab8a8ccad4b505bedfa1783f98fbf93685eee4ff0e165ce2167f13ec71937cde9a50b47846f59e537082d502b118d0431159cf952f089febd4a779df816b7e94e30 -->

goerz commented:
Note that you can make changes to your package but leave the version number unchanged. When you retrigger registration from the changed commit, it will update the existing PR instead of creating a new one. Of course, this means you should never create a git tag before the registration for the new version has actually gone through. Indeed, [you are using TagBot](https://github.com/subhk/BiGSTARS.jl/blob/main/.github/workflows/TagBot.yml) (as you should), and that automatically creates tags after registration. That would be the normal workflow for Julia packages. I would strongly recommend that you delete any `v2.0.0` and `v2.0.1` tags in your repository, and just amend the `v2.0.0` registration.

JuliaRegistrator committed:
New package: BiGSTARS v2.0.2

UUID: 55e6fa54-20b2-4a8e-ba84-e72db1c430e6
Repo: https://github.com/subhk/BiGSTARS.jl.git
Tree: 227344b7b98afd94ed05dddf3c8a3d9458d7f2a2

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

subhk commented:
Thanks. now I've v2.0.0.
